pkg/controller/controllertest/util.go

```diff
@@ -135,8 +147,25 @@ func (f *FakeInformer) AddEventHandlerWithResyncPeriod(handler cache.ResourceEve
 	return nil, nil
 }
 
-// RemoveEventHandler does nothing.  TODO(community): Implement this.
+// RemoveEventHandler does nothing.
```

Comment:
```suggestion
// RemoveEventHandler removes an EventHandler to the fake Informers.
```
nit: sorry missed it the last time